packages/nodes-base/nodes/HttpRequest/V3/HttpRequestV3.node.ts

```diff
@@ -1129,10 +1135,25 @@ export class HttpRequestV3 implements INodeType {
 						itemIndex,
 					) as string;
 					this.helpers.assertBinaryData(itemIndex, inputDataFieldName);
-					requestOptions.body = await this.helpers.getBinaryDataBuffer(
-						itemIndex,
-						inputDataFieldName,
-					);
+
+					let uploadData: Buffer | Readable;
+					let contentLength: number;
+
+					const itemBinaryData = items[itemIndex].binary![inputDataFieldName];
```

Comment:
Delete line 1137, and change this to 

```suggestion
					const itemBinaryData = this.helpers.assertBinaryData(itemIndex, inputDataFieldName);
```